src/system/ui/pages/post/index.js

```diff
@@ -16,6 +16,50 @@ export const Post = ( { location, match, history } ) => {
 	const [ passwordVisible, setPasswordVisible ] = useState( 'protected' === item.visibility )
 	const parentOptions = useParentOptions( item.type )
 	const wpRest = getWpRest()
+	const [ featureThumbnail, setfeatureThumbnail ] = useState( item.thumbnail )
+	const [ thumbData, setthumbData ] = useState( {} )
+	const [ hasUpdateimg, sethasUpdateimg ] = useState( false )
+	const [ removeThumbnail, setremoveThumbnail ] = useState( false )
```

Comment:
These variables should be camelCase.

Reply:
I am confused is it not in camelCase? like featureThumbnail has a T capital.

Reply:
Some of them are, but others like `setfeatureThumbnail` should be `setFeatureThumbnail`. Typically in hook setters, the word after `set` is uppercase.

Reply:
Okay, Understood and updated.